docs/tools/vdb_table/data/nucliadb.json

```diff
@@ -26,127 +26,127 @@
     "source_url": "",
     "comment": ""
   },
-  "github_stars": 497,
-  "vector_launch_year": 0,
+  "github_stars": 537,
+  "vector_launch_year": 2021,
   "metadata_filter": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/docs/internal/SEARCH.md",
     "comment": ""
   },
   "hybrid_search": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/docs/internal/SEARCH.md",
     "comment": ""
   },
   "facets": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/docs/internal/SEARCH.md",
     "comment": ""
   },
   "geo_search": {
-    "support": "",
+    "support": "none",
     "source_url": "",
     "comment": ""
   },
   "multi_vec": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/README.md",
```

Comment:
Could you provide an example of an index where multiple fields are embedded?
I couldn't figure it out from your link or https://docs.nuclia.dev/docs/guides/sdk/python-sdk/read/#get-embeddings

Reply:
reverting